pkgs/applications/audio/gwc/default.nix

```
@@ -0,0 +1,52 @@
+{
+  stdenv
```

Personal preference:

```suggestion
{ stdenv
,
```

---

OK, done.